server/sonar-web/src/main/js/apps/issues/facets/mode-facet.js

```diff
@@ -17,20 +17,20 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
+import $ from 'jquery';
 import _ from 'underscore';
 import BaseFacet from './base-facet';
 import Template from '../templates/facets/issues-mode-facet.hbs';
 
 export default BaseFacet.extend({
   template: Template,
 
-  events: {
-    'change [name="issues-page-mode"]': 'onModeChange'
-  },
-
-  onModeChange () {
-    const mode = this.$('[name="issues-page-mode"]:checked').val();
-    this.options.app.state.updateFilter({ facetMode: mode });
+  toggleFacet (e) {
+    const isCount = $(e.currentTarget).is('[data-value="count"]');
+    // $(e.currentTarget).toggleClass('active');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3ACommentedCode)